src/app/ui/TimedToasts/WarningTimedToast.tsx

```diff
@@ -0,0 +1,32 @@
+interface WarningTimedToastProps {
+  idToast: string
+  message: string
+}
+
+const WarningTimedToast: React.FC<WarningTimedToastProps> = ({ idToast, message }) => {
```

Comment:
Is this method linked to the warning type? It would be good if other types of messages such as error messages could be configured, so that they can be parameterized with just one object.